packages/jest-config/src/utils.js

```diff
@@ -59,27 +59,29 @@ const _replaceRootDirInPath = (rootDir: string, filePath: Path): string => {
   );
 };
 
+const _replaceRootDirInObject = (rootDir: string, config: any): Object => {
+  if (config instanceof RegExp) {
```

Comment:
I actually feel like we may be able to remove this in a follow-up PR. It can't be a regex at this point, can it?